integration-tests/Cargo.toml

```diff
@@ -12,12 +12,10 @@ workspace = true
 
 [dependencies]
 
+[dev-dependencies]
```

Comment:
Should we delete this crate ? Its a no op at the moment

Reply:
Yah sounds good